drivers/modem/CMakeLists.txt

```diff
@@ -1,2 +1,3 @@
 zephyr_sources_ifdef(CONFIG_MODEM_RECEIVER modem_receiver.c)
+zephyr_sources_ifdef(CONFIG_MODEM_SHELL	   modem_shell.c)
```

Comment:
\t seems to be unnecessary here as the surrounding lines do not have that either